### DISCUSSION TITLE
Books Grouping Feature

### DISCUSSION THREAD

**Discussion by ghost** [PaulWoitaschek/Voice#1918] [Ideas]
I have some collection like Harry Potter, Game of Thrones and i want to group them. (like i click on collection and it goes in and shows all parts)

And group cover feature would also be nice, like i set Harry Potter cover for whole collection.

**Comment by mesonium** [comment]
~Duplicate of https://github.com/PaulWoitaschek/Voice/discussions/1918.~ See below.

**Reply by polyzen** [reply]
Duplicate of #1314*

**Reply by mesonium** [reply]
Thank you for the correction.

**Reply by ghost** [reply]
What a difference, developers didn't give a fuck to suggestion from 7 years back, nor this one.

**Reply by PaulWoitaschek** [reply]
Oh that's rude! Learn developing and implement it in your free time yourself!
